### DISCUSSION TITLE
Trojan security threat with download

### DISCUSSION THREAD

**Discussion by niyaz-hawaldar** [GitTools/GitVersion#2897] [General]
Hi 
I am trying to download  gitversion-win-x64-5.7.0.tar.gz
getting trojan threat "Trojan:Win32/Spusint.F!cl" with status "severe".

**Comment by HHobeck** [comment]
Do you get this security warning for https://github.com/GitTools/GitVersion/releases/tag/6.0.0-beta.2 as well?

**Comment by HHobeck** [comment]
This is an open source project and we have tools to detect code which might be a security risk. I'm going to close this issue. Please report a bug if this problem remains. Thank you very much.
